src/scene/constants.js

```diff
@@ -183,11 +183,11 @@ export var LAYERID_UI = 4;
 export var LIGHTTYPE_DIRECTIONAL = 0;
 /**
  * @constant
- * @name pc.LIGHTTYPE_POINT
+ * @name pc.LIGHTTYPE_OMNI
```

Comment:
it seems you have removed pc.LIGHTTYPE_POINT? Unless that was moved elsewhere. This probably breaks existing scripts? Test with engine tests, few of them would use it.

Reply:
The API for creating lights uses the string "point" - which is aliased to LIGHTTYPE_OMNI - I didn't find anywhere that uses LIGHTTYPE_POINT directly

Reply:
Unfortunately pc.LIGHTTYPE_POINT is part of the public API - so I'll put it back in - and mark pc.LIGHTTYPE_OMNI as private for now. Later we should mark pc.LIGHTTYPE_POINT as deprecated but still around for legacy support

Reply:
good point. Either way, we need to keep it alive, as its public and likely some people use it. Maybe add it to deprecated.js, something similar to 
`export var RIGIDBODY_TYPE_STATIC = BODYTYPE_STATIC;`

Reply:
done